modules/nw-egress-ips-about.adoc

```diff
@@ -44,15 +44,15 @@ Support for the egress IP address functionality on various platforms is summariz
 | Platform | Supported
 
 | Bare metal | Yes
-| VMware vSphere | Yes
-| {rh-openstack-first} | Yes
-| Amazon Web Services (AWS) | Yes
-| Google Cloud Platform (GCP) | Yes
-| Microsoft Azure | Yes
-| {ibm-z-name} and {ibm-linuxone-name} | Yes
-| {ibm-z-name} and {ibm-linuxone-name} for {op-system-base-full} KVM | Yes
-| {ibm-power-name} | Yes
-| Nutanix | Yes
+| VMware vSphere | No
+| {rh-openstack-first} | No
+| Amazon Web Services (AWS) | No
+| Google Cloud Platform (GCP) | No
+| Microsoft Azure | No
+| {ibm-z-name} and {ibm-linuxone-name} | No
+| {ibm-z-name} and {ibm-linuxone-name} for {op-system-base-full} KVM | No
+| {ibm-power-name} | No
+| Nutanix | No
```

Comment:
All these platforms support egress IPs for the primary host network. They do not for secondary host networks, except for baremetal platform which is supported.